kernel/aster-nix/src/process/credentials/capabilities.rs

```diff
@@ -52,6 +52,8 @@ bitflags! {
     }
 }
 
+pub const CAP_LAST_CAP: u64 = 40; // Number of the last capability (CAP_CHECKPOINT_RESTORE)
```

Comment:
Defining this const is unreadable. Reading from this const and the module along, it is unobvious what it is the meaning of it. Instead, defining it as a method of `CapSet` gives it a context for the reader to understand its meaning, thus making the code more readable and maintainable.

```rust
impl CapSet {
    /// The most significant bit in a 64-bit `CapSet` that may be set to represent a Linux capability.
    pub fn most_significant_bit() -> u8 {
        // CHECKPOINT_RESTORE is the Linux capability with the largest numerical value
        40
    }
}
```

Reply:
Done.